SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -242,12 +242,16 @@ void USpatialWorkerConnection::ConnectToLocator()
 	FTCHARToUTF8 PlayerIdentityTokenCStr(*LocatorConfig.PlayerIdentityToken);
 	FTCHARToUTF8 LoginTokenCStr(*LocatorConfig.LoginToken);
 
-	Worker_Alpha_LocatorParameters LocatorParams = {};
+	Worker_LocatorParameters LocatorParams = {};
+	FString ProjectName;
+	FParse::Value(FCommandLine::Get(), TEXT("projectName"), ProjectName);
```

Comment:
Gets the project name from the command line, since it's available there. Is there a connection flow that uses the locator that does not get the project name in the command line?